### PR TITLE
Update README for Clarity 

### PR DESCRIPTION
**Pull request by Romero027**
## Why are these changes needed?

The README page is somewhat confusing in certain sections. This PR aims to make the README page more readable.

### CODE REVIEW

**Comment by Romero027** [comment]
@crazyboycjr can you take a look?

File: README.md [hunk 1]
```diff
@@ -30,62 +30,57 @@ $ git clone git@github.com:phoenix-dataplane/phoenix.git --recursive
...
 
-For inter-host RPC in the following `rpc_hello` example, you may need at
+<!-- For inter-host RPC in the following `rpc_hello` example, you may need at
 least two machines. However, you can still run the client and server on the same
 machine, communicating through the same instance of phoenixos.
-Choose whichever test scenario that is most suitable for you.
+Choose whichever test scenario that is most suitable for you. -->
 
-To begin with, make sure exactly one instance of phoenixos is running on all servers,
+Ensure that exactly one instance of PhoenixOS is running on all servers.
```
Comment (crazyboycjr) [inline README.md:73]: I just realized that this sentence could be interpretted differently. Maybe change it to:
Ensure that exactly one instance of PhoenixOS is running on each server.

File: README.md [hunk 2]
```diff
@@ -96,20 +91,20 @@ with a prebuilt set of phoenix crates. This is currently done by
...
 1. (Recommended) To start the applications on multiple machines, we prepare
 a launcher for this job.
 ```bash
 $ cd ../../benchmark
 Follow the README under benchmark directory and update config.toml
 $ cargo rr --bin launcher -- --benchmark benchmark/rpc_hello.toml
-```
+``` -->
 
 2. Alternatively, you can run the examples manually by
```
Comment (crazyboycjr) [inline README.md:103]: Remove this `2. Alternatively` and maybe add a pointer to the [`launcher`](https://github.com/phoenix-dataplane/phoenix/tree/main/benchmark) at the end of this section.

**Review by crazyboycjr — CHANGES_REQUESTED** [review]
Thank you for the changes!

**Review by crazyboycjr — APPROVED** [review]
Looks great to me!

**Pull request merged**